cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v14
+  name: heapster-v15
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v14
+    version: v15
```

Comment:
That will lead to more salt ***... The version numbers will be eliminated
once we move to deployments.


On Thu, Mar 3, 2016 at 5:13 PM, Tim St. Clair <notifications@github.com>
wrote:

> In cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml
> <https://github.com/kubernetes/kubernetes/pull/22492#discussion_r54977218>
> :
>
> >    namespace: kube-system
> >    labels:
> >      k8s-app: heapster
> > -    version: v14
> > +    version: v15
>
> Consider making v15 and v0.20.0-alpha11 templated parametrers (like
> heapster_memory) so we only need to change it in 1 place
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22492/files#r54977218>.
>